### PR TITLE
chore (refs DPLAN-11413): replace v-tooltip directive with DpContextu…

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://demoseurope.youtrack.cloud/issue/DPLAN-11413

- replace v-tooltip directive with DpContextualHelp component.
- to unification contextual.help.

### CODE REVIEW

File: client/js/components/procedure/admin/AuthorizedUsersList.vue [hunk 1]
```diff
@@ -102,10 +102,9 @@
           aria-hidden="true" />
         {{ Translator.trans('export') }}
       </a>
-      <i
-        class="fa fa-question-circle inline-block align-top u-ml-0_25 u-mt-0_125"
-        :aria-label="Translator.trans('contextual.help')"
-        v-tooltip="Translator.trans('consultation.export.bulk.letter.explanation')" />
+      <dp-contextual-help
+        class="inline-block align-top u-ml-0_25 u-mt-0_125"
```
Comment (spiess-demos) [inline client/js/components/procedure/admin/AuthorizedUsersList.vue:106]: I can imagine that `align-top` is no longer needed.

**Review by spiess-demos — COMMENTED** [review]
Nice! Only a few questions.

File: client/js/components/procedure/admin/AuthorizedUsersList.vue (hunk 1 above)
Comment (spiess-demos) [reply client/js/components/procedure/admin/AuthorizedUsersList.vue:106]: ok, i see this component is not in use within any project at the moment. in this case we may leave it as it is.

Comment (ahmad-demos) [reply client/js/components/procedure/admin/AuthorizedUsersList.vue:106]: @spiess-demos 
`DpSurveyStatus` or `AuthorizedUsersList`

Comment (ahmad-demos) [reply client/js/components/procedure/admin/AuthorizedUsersList.vue]: Done in https://github.com/demos-europe/demosplan-core/pull/2955/commits/b2a9d5597cbcb27fb2155c1cee68774d3fe5d0a6 for `AuthorizedUsersList`

File: client/js/components/procedure/DpSurveyStatus.vue [hunk 2]
```diff
@@ -17,9 +17,9 @@
         {{ surveyStatusHint }}
       </p>
     </label>
-    <i
-      class="fa fa-question-circle u-pt-0_25 inline-block float-right"
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply client/js/components/procedure/DpSurveyStatus.vue:21]: https://github.com/demos-europe/demosplan-core/pull/2955#discussion_r1562175247

File: client/js/components/procedure/admin/AuthorizedUsersList.vue (hunk 1 above)
Comment (spiess-demos) [reply client/js/components/procedure/admin/AuthorizedUsersList.vue]: DpSurveyStatus is not used at the moment i guess.

**Review by spiess-demos — APPROVED** [review]
lgtm (code review)

**Pull request merged**